doc/pod1/cellcc_config.pod

```diff
@@ -103,12 +103,40 @@ mysql database called "cellcc" on host db.example.com using port 3306.
 
 The username and password to use when connecting to the B<db/ro/dsn> database.
 
+=item B<db/ro/options>
+
+This optional directive can be used to add / alter the settings to be used when
+contacting the database. For example, the admin can specify the timeout to use
```

Comment:
the current convention in the manpage is to say "you" instead of "the admin". I think talking directly to the user is fine